packages/terra-application-utility/tests/jest/UtilityMenu.test.jsx

```diff
@@ -100,4 +100,42 @@ describe('UtilityMenu', () => {
     const wrapper = shallow(component);
     expect(wrapper).toMatchSnapshot();
   });
+
+  it('should render with a user specified role', () => {
+    const role = 'menu';
+    const component = (
+      <IntlProvider locale={locale} messages={messages}>
+        <HeaderUtilityMenu
+          initialSelectedKey="key"
+          menuItems={config}
+          onChange={mockOnChange}
+          onDisclose={mockOnDisclose}
+          variant={variant}
+          role={role}
+          id="test"
+        />
+      </IntlProvider>);
+    const wrapper = shallow(component);
+    expect(wrapper.prop('role')).toBe(role);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should render with a default role', () => {
+    const defaultRole = 'navigation';
+    const component = (
+      <IntlProvider locale={locale} messages={messages}>
+        <HeaderUtilityMenu
+          initialSelectedKey="key"
+          menuItems={config}
+          onChange={mockOnChange}
+          onDisclose={mockOnDisclose}
+          variant={variant}
+          role={defaultRole}
```

Comment:
```suggestion
          menuRole={defaultRole}
```

Reply:
[Changed to menurole](https://github.com/cerner/terra-framework/pull/501/commits/131aaf225ec1730dafcfecf63f4d79d0857c6bfa)

Reply:
update to camelCase - `menuRole`

Reply:
When I update this to camel case It throws warning saying : 
<img width="1671" alt="screen shot 2019-03-01 at 8 44 10 pm" src="https://user-images.githubusercontent.com/12869809/53647260-27487680-3c63-11e9-84b6-99b13e28f0db.png">